### DISCUSSION TITLE
Why am I getting "This Serverless Function has timed out." Error

### DISCUSSION THREAD

**Discussion by ChrisB007** [nextauthjs/next-auth#2827] [Help]
Hello guys, I need your help to resolve this issue when trying to access the protected page( dashboard) on my app. I am getting this error "This Serverless Function has timed out." 
<img width="536" alt="Screen Shot 2021-09-23 at 2 04 53 PM" src="https://user-images.githubusercontent.com/33075219/134560313-e9dc2a74-23b8-45b2-8164-facb8b9c0190.png">

What is more pressing is that if I refresh the page, then the login provider shows up. Can you help?

**Comment by balazsorban44** [comment]
Hard to guess, do you have a reproduction? Sounds like you forgot to end the response and the endpoint is hanging.
